maintainers/working_groups/learning_journey/documentation-survey.md

```diff
@@ -236,13 +236,13 @@ To better navigate the material and judge its relevance, every entry should prov
 
 ### Nix
 
-- https://nixos.org/manual/nix/stable/command-ref/command-ref.html
+- https://nix.dev/manual/nix/2.18/command-ref/command-ref.html
```

Comment:
Do these links have to hardcode the domain name? That feels weird, makes them display differently and break the deploy preview

Reply:
As described in the commit/PR message: unfortunately yes. Unless @yukiisbored has another idea, we have exhausted our immediately available Sphinx-fu with regard to alternatives.